### PR TITLE
Task f6 auction feature

### PR DESCRIPTION
**Pull request by bransometan**
Added the Front End Feature for Auction and connected to Firestore database.

### CODE REVIEW

File: app/Screens/Splashscreen.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {View, Image, StyleSheet} from 'react-native';
+import {View, Image, StyleSheet, AsyncStorage} from 'react-native';
```
Comment (anshumaantgi) [inline app/Screens/Splashscreen.js:2]: Can Remove this as it is never used in Code !

File: app/Screens/MyProfilepage.js [hunk 2]
```diff
@@ -1,17 +1,85 @@
...
+                // Do work here
+                //console.log('@sutd.edu.sg');
+                setUseruni('Singapore University of Technology & Design (SUTD)');
+                break;
+            case "SUSS" :
+                // Do work here
+                //console.log('@suss.edu.sg');
+                setUseruni('Singapore University of Social Sciences (SUSS)');
+                break;
+            case "GMAIL" :
```
Comment (anshumaantgi) [inline app/Screens/MyProfilepage.js:61]: Can Also remove this as it was only used for debugging!

File: app/Screens/Auction1.js [hunk 3]
```diff
@@ -1,18 +1,148 @@
...
+                // Do work here
+                //console.log('@sutd.edu.sg');
+                setUseruni('Singapore University of Technology & Design (SUTD)');
+                break;
+            case "SUSS" :
+                // Do work here
+                //console.log('@suss.edu.sg');
+                setUseruni('Singapore University of Social Sciences (SUSS)');
+                break;
+            case "GMAIL" :
```
Comment (anshumaantgi) [inline app/Screens/Auction1.js:106]: Similarly Here

**Review by anshumaantgi — CHANGES_REQUESTED** [review]

File: app/Screens/Auction1.js (hunk 3 above)
Comment (bransometan) [reply app/Screens/Auction1.js:106]: Noted, next merge in the development branch will be removed.

File: app/Screens/Splashscreen.js (hunk 1 above)
Comment (bransometan) [reply app/Screens/Splashscreen.js:2]: Noted, next merge in the development branch will be removed.

File: app/Screens/MyProfilepage.js (hunk 2 above)
Comment (bransometan) [reply app/Screens/MyProfilepage.js:61]: Noted, next merge in the development branch will be removed.

**Review by bransometan — COMMENTED** [review]
Noted, next merge in the development branch will be removed.

**Pull request merged**